gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 [versions]
-pulsar = "2.11.0"
+pulsar = "3.0.0"
```

Comment:
In thinking about versioning the project that will deliver this change, I noticed that [build.gradle](https://github.com/apache/pulsar-client-reactive/blob/main/gradle.properties#L20) is at `0.2.1-SNAPSHOT`. Shouldn't this be `03.1-SNAPSHOT`? I think we may have missed a post-release step on the `0.3.0` release?

Reply:
You're right. Feel free to open a PR for it :D